src/manifolds/MetricManifold.jl

```diff
@@ -194,22 +186,7 @@ Compute the Gaussian curvature of the manifold `M` at the point `x`.
 """
 gaussian_curvature(M::MetricManifold, p; kwargs...) = ricci_curvature(M, p; kwargs...) / 2
 
-function get_basis(M::MMT, p, B::ArbitraryOrthonormalBasis) where {MMT<:MetricManifold}
```

Comment:
Are you sure this disambiguation is no longer needed?

Reply:
It's already done on the `AbstractDecorator` level for _all_ decorators already, now.

Reply:
That great 👍 .